.travis.yml

```diff
@@ -2,6 +2,11 @@ language: node_js
 node_js:
  - "4.4.7"
 before_install:
+- git config --global user.name "Travis CI"
+- git config --global user.email "travis@travis-ci.org"
+- git remote add origin_ssh git@github.com:alphagov/govuk_elements.git
+- openssl aes-256-cbc -K $encrypted_85ebe8034b89_key -iv $encrypted_85ebe8034b89_iv -in .travis/govuk_elements.enc -out ~/.ssh/id_rsa -d
```

Comment:
If this is generated by Travis CLI can we add a comment about this (and potentially how it can be regenerated in the future)?